includes/class-wc-stripe-settings.php

```
@@ -0,0 +1,564 @@
+	public function get_test_refresh_token(): string {
+		return $this->settings['test_refresh_token'] ?? '';
+	}
+
+	/**
+	 * Get the legacy method order.
+	 *
+	 * @return array<int, string>
+	 */
+	public function get_stripe_legacy_method_order(): array {
+		return $this->settings['stripe_legacy_method_order'] ?? [];
+	}
```

PHPMD (2.15.0)

[error] The method get_stripe_legacy_method_order is not named in camelCase. (undefined)

(CamelCaseMethodName)